### ISSUE TITLE
Numerous assets/zarrs in 000108:  asset is currently being validated, please wait.

### ISSUE THREAD

**Issue by yarikoptic** [dandi/dandi-archive#1473]
![image](https://user-images.githubusercontent.com/39889/217280924-329834c7-9777-478b-a3f4-8712ab0f3b74.png)

and it seems that the vast majority of those >900 warnings is about "being validated". 

Something is really not right there and I have not found a relevant issue. https://github.com/dandi/dandi-archive/issues/567 was about related but triggered by testing in dandi-cli which was worked around in https://github.com/dandi/dandi-cli/pull/957 .

**Comment by AlmightyYakob** [comment]
> In the meantime, I can reset this dandiset and ping here when it's done.

@satra This has been done.

**Comment by satra** [comment]
looks like several current issues are checksum related. i think checksum generation may also need to be rerun, and if rerunning checksums doesn't automatically rerun validation, then that should happen as well.

**Comment by waxlamp** [comment]
@AlmightyYakob, I still see some "being validated" errors on 108. Is our strategy here still to rerun validation (and/or checksums) until those messages are gone, or should we close this in favor of #1479 for a more comprehensive solution? I would say let's solve the problems with 108 if that will unblock the 108 owners on something they're currently trying to accomplish.

**Comment by AlmightyYakob** [comment]
Sure, I can directly fix 108, and continue on with the full solution.

**Comment by satra** [comment]
i would add 26 to this list. not zarr but in similar "being validated" state. in fact i don't know how many are in that state. it may be worthwhile to do a db check on which dandisets have assets in "being validated" state.

**Comment by AlmightyYakob** [comment]
@satra This has been taken care of for both `000108` and `000026`. Closing this in favor of #1479, as this tracks the real issue.
